modules/Core/hooks/DiscordFormatterHook.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/*
+ *  Made by Partydragen
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.0 pre-13
+ *
+ *  Discord formatter hook
+ */
+
+class DiscordFormatterHook extends HookBase {
+
+    public static function format(array $params = []): array {
+        $data = $params['data'];
+        $format = $params['format'];
+
+        if ($data['event'] == 'registerUser') {
```

Comment:
It would be great if these weren't hard coded here. 

Can we utilize the event you made in this PR within the core module to format them?